src/lib.rs

```diff
@@ -211,6 +211,14 @@ pub fn restart_app_if_necessary(app_id: AppId) -> bool {
     unsafe { sys::SteamAPI_RestartAppIfNecessary(app_id.0) }
 }
 
+/// When you are done using the Steamworks API you should call
+/// SteamAPI_Shutdown to release the resources used by your application
+/// internally within Steam. You should call this during process
+/// shutdown if possible.
+pub fn shutdown() {
```

Comment:
IMO this should be on the Drop implementation for Client instead to ensure that shutdown is called in any graceful shutdown of the process.